tests/gtest/avifcolrtest.cc

```diff
@@ -52,7 +53,7 @@ TEST(TransferCharacteristicsTest, RoundTrip) {
     }
 
     if (tc == AVIF_TRANSFER_CHARACTERISTICS_LOG100) {
-      EXPECT_EQ(min_linear, kTransferLog100Threshold / 2.0f);
+      EXPECT_NEAR(min_linear, kTransferLog100Threshold / 2.0f, 1e-7);
```

Comment:
Nit: The constants in line 56 and line 109 can have the `f` suffix.

Reply:
That matters because it is all about having bad float computations here... Done in https://github.com/AOMediaCodec/libavif/pull/1888